packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -228,6 +241,10 @@ export class WorkspaceCommandContribution implements CommandContribution {
                 }
             }
         }));
+        registry.registerCommand(WorkspaceCommands.OPEN_ROOT_CONFIG, {
+            isEnabled: () => this.workspaceService.opened && this.preferences['workspace.supportMultiRootWorkspace'],
```

Comment:
`this.workspaceService.opened && this.preferences['workspace.supportMultiRootWorkspace']` seems to pop up a lot. How about `workspaceService.supportMultiRoot` readonly property?